docs/diagrams/ShowRelatedActivityDiagram.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+
+
+title Activity: showrelated command
+start
+:User executes showrelated command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+switch ()
+case ([One valid number (Integer and greater than 0) provided])
+    :Return list of related profiles;
+case ([Else])
+    :Return error message;
+endswitch
```

Comment:
Would you want to add a case for [One valid number but id does not exist], but still returns empty list of related profiles?